### DISCUSSION TITLE
Using npx cy2 in conjunction with cypress-ntlm

### DISCUSSION THREAD

**Discussion by TheDMon** [sorry-cypress/sorry-cypress#583] [Q&A] [answered]
I am exploring opportunities to integrate sorry cypress with our cypress project which performs tests on intranet hosted corporate app. We are using cypress-ntlm-auth plugin to pass windows authentication to run tests. So, we do npx cypress-ntlm run at this moment.

I searched over the net but did not find anything that would help me to understand how I can use npx cy2 along with npx cypress-ntlm. Any help would be really appreciated.

Thanks in advance!

**Comment by agoldis** [comment, marked as answer]
Try invoking `patch` method in a script: https://github.com/sorry-cypress/cy2#patch-cypress
Or just configure cypress manually: https://docs.sorry-cypress.dev/cypress-agent/configuring-cypress-agent

**Reply by TheDMon** [reply]
Thanks @agoldis!

While I'm certainly able to integrate sorry cypress dashboard with manually changing the api_url on my local, unfortunately I cannot script update it in the app.yml file on agent due permission issue. So, I'm looking at the patch method that you suggested; but I am bit lost as to where do I put the patch call in my cypress project to update the api_url programmatically. Should it be place in index.ts under plugin folder? 

Forgive my ignorance and thanks for being awesome!

**Reply by TheDMon** [reply]
Hey, I think I figured it out. 
I've created a cy2patch.js file in my project with code that is in [here](https://github.com/sorry-cypress/cy2#patch-cypress) and created an entry under script in my package.json like below to call it for I do cypress-ntlm run.

"scripts": {
    "cy2patch": "node cy2patch.js"
  }
